### ISSUE TITLE
Implement detailed watchlist view

### ISSUE THREAD

**Issue by Martenere** [Watchlist-project/movie-watchlist-frontend#20]
**screen mockups (can be done on pen and paper for speed)**
 ![Image](https://github.com/Watchlist-project/movie-watchlist-frontend/assets/78728730/e32559df-dc7c-4c00-bc03-1c167e1c269e)

**for each screen: tell me the URL router path in the app**
Path: _watchlists/:id_

**tell me if any fetch requests must be done (and to what API endpoint)**
  - Delete watchlist API call at endpoints
    - Delete API not implemented in API endpoints yet TODO
      - Delete call at *watchlists/:id*
  - Put API call to update watchlist details
![Image](https://github.com/Watchlist-project/movie-watchlist-frontend/assets/153117666/8ce7710a-18c0-46bd-aec9-ac6a2072624e)

**tell me any particular state that needs to be stored**
Fetch request is done by jotai atom _watchlistState_ and the selected watchlist is stored in atom _currentlyEditingState_
So in short use _currentlyEditingState_ to get watchlistData

**list of actions the user can perform (clicks, typing in forms, etc...)**
  - Click on watchlist name in navbar to navigate to watchlist/:id
  - 3 dot menu inside of detailed view which gives acess to 
    - Edit opens a [Modal form](https://mantine.dev/core/modal/)
    - Delete watchlist

**list of how I redirect / navigate to other pages from this screen**
Using navbar for main navigation

*Quick planning before issue was created*
![Image](https://github.com/Watchlist-project/movie-watchlist-frontend/assets/78728730/e2252ab8-e258-4f72-8ead-5f55d2b68f83)

Modal view, inspiration from spotify
![Image](https://github.com/Watchlist-project/movie-watchlist-frontend/assets/153117666/70b44531-f65b-4120-8ae9-be2c386f5e4e)

**Comment by LinusWillmont** [comment]
Merged this branch for other team members to start working on other thing related
Reopened to finish functionality
